cranelift/codegen/src/isa/riscv64/inst.isle

```diff
@@ -1913,6 +1914,11 @@
 (decl intcc_to_extend_op (IntCC) ExtendOp)
 (extern constructor intcc_to_extend_op intcc_to_extend_op)
 
+
+;; parameters are in_ty out_ty input is_signed  
+(decl normalize_fcvt_from_float (Type Type Reg bool) Reg)
```

Comment:
Can we rename this to something like `clamp_*`? Unlike the normalize functions this one actually changes the value of the register.